.github/workflows/release.yml

```diff
@@ -71,7 +71,10 @@ jobs:
         with:
           context: .
           file: ./docker/Dockerfile.${{ matrix.service }}
-          tags: vesoft/nebula-${{ matrix.service }}:${{ steps.tagname.outputs.tag }},vesoft/nebula-${{ matrix.service }}:latest
+          tags: |
+            vesoft/nebula-${{ matrix.service }}:${{ steps.tagname.outputs.tag }}
+            vesoft/nebula-${{ matrix.service }}:${{ steps.tagname.outputs.majorver }}
+            vesoft/nebula-${{ matrix.service }}:latest
```

Comment:
suppose we release v3.1.0 and v2.7.1 at the same, it may be confused for `latest` version.
how about just mark the highest version as latest?

in this case, v3 should be the latest, but v2 should not.

Reply:
Fixed